### ISSUE TITLE
Many dead links with the README.md

### ISSUE THREAD

**Issue by milekey** [Hexworks/zircon#79]
The README.md has a lot of dead links though I want to peruse about this nice library.

**Comment by adam-arold** [comment]
Yep, the documentation definitely needs some polish. I'll fix it next week probably on Wednesday or Thursday.

**Comment by milekey** [comment]
Thank you for your quick reply! I'm looking forward to.

**Comment by adam-arold** [comment]
I fixed the links and code examples also!

**Comment by milekey** [comment]
Thank you!

I found a few more dead links on another wiki page: Resource Handling. Sorry, again.

**Comment by adam-arold** [comment]
Whoops, I'll take a look. I'll definitely need to create some automatism for this. I always bump into these dead links. Are you an active user of Zircon?

**Comment by milekey** [comment]
I'm very new to your library. So, I am reading those documents.

**Comment by adam-arold** [comment]
I'd suggest for you to use the latest snapshot from [jitpack](https://jitpack.io/p/Hexworks/zircon#download)
